.vscode/settings.json

```diff
@@ -31,6 +31,11 @@
   },
   "[json]": {
     "editor.formatOnSave": true,
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
-  }
-}
+    "editor.defaultFormatter": "vscode.json-language-features"
+  },
+  "python.testing.pytestArgs": [
+    "."
+  ],
+  "python.testing.unittestEnabled": false,
+  "python.testing.pytestEnabled": true
+}
```

Comment:
Could we change this back to what was there earlier. In case we wanna change this we can create a new PR as this will effect a lot of people

Reply:
This has been fixed